packages/ui-new/src/utils/ThemeUtil.ts

```diff
@@ -1,13 +1,37 @@
 import { css, unsafeCSS } from 'lit'
 import { getW3mThemeVariables } from '@reown/appkit-common'
 import type { ThemeVariables, ThemeType } from '@reown/appkit-common'
+import { ThemeHelperUtil } from './ThemeHelperUtil.js'
 
 // -- Utilities ---------------------------------------------------------------
+let apktTag: HTMLStyleElement | undefined = undefined
 let themeTag: HTMLStyleElement | undefined = undefined
 let darkModeTag: HTMLStyleElement | undefined = undefined
 let lightModeTag: HTMLStyleElement | undefined = undefined
 
-export function initializeTheming(themeVariables?: ThemeVariables, themeMode?: ThemeType) {
+/* Temprorary for now */
+const fonts = {
```

Comment:
Possibly we are going to change this URL, let's not forget about it

Reply:
Yeah we will definitely change the font URL, i just use these for now for development 🙏

Reply:
Actually let me add `@TODO` comment so we don't forget.